test-two/src/index.js

```diff
@@ -4,23 +4,29 @@ import { getUsersForOrganisation } from "./services/github";
 const baseUrl = "https://api.github.com/";
 const organisationId = "google";
 const gitHubApiAuthToken = ""; // add your GitHub API OAuth key here to increase request limit
+var pageNumber;
```

Comment:
Shouldn't we be using const or let?

Reply:
I thought const  was only used for variables that can't be reassigned and I didn't think let would of scooped properly.